packages/api/resolvers/mutations/accounts/verifyEmail.js

```diff
@@ -1,5 +1,19 @@
-import callMethod from '../../../callMethod';
+import {
+  accountsServer,
+  accountsPassword,
+  dbManager,
+} from 'meteor/unchained:core-accountsjs';
 
 export default async function verifyEmail(root, { token }, context) {
-  return callMethod(context, 'verifyEmail', token);
+  const user = await dbManager.findUserByEmailVerificationToken(token);
+  await accountsPassword.verifyEmail(token);
```

Comment:
Could this possibly return false and what if it does?

Reply:
Given a false token, it definitely would. TBH, I've thought about wrapping all the mutations of `packages/api/resolvers/mutations/accounts` with try/catch but was unsure of what type of errors I should be throwing. Like vague ones tailored to each request i.e 'Change password request failed` or what. What do you think?